lib/widget/stateful/screens/employee_info_screen.dart

```diff
@@ -0,0 +1,196 @@
+    }).catchError((error) {
+      Message.giveMessage(context, ErrorHandler.handleError(error));
+    });
+  }
+
+  void _doLogout() {
+    Util.logoutUser().then((_) {
+      Message.giveMessage(context, "Logout successful !");
+      Navigator.pushAndRemoveUntil(
+        context,
+        MaterialPageRoute(builder: (context) => new WelcomeScreen()),
+        (Route<dynamic> route) => false,
+      );
+    }).catchError((error) {
+      Message.giveMessage(context, ErrorHandler.handleError(error));
+    });
+  }
+
+  Widget build(BuildContext context) {
+    var assetImage = new AssetImage("assets/images/sunglasses-c.png");
+    var image = new Image(image: assetImage, height: 50.0, width: 330.0);
+    return MaterialApp(
+        home: DefaultTabController(
+            length: 1,
+            child: Scaffold(
+              key: scaffoldKey,
+              appBar: AppBar(
+                  backgroundColor: Color(0xFF1c1a34),
+                  leading: Builder(builder: (BuildContext context) {
+                    AssetImage assetImage = AssetImage('assets/images/org.png');
+                    Image image =
+                        Image(image: assetImage, width: 19.0, height: 17.0);
+                    return Container(child: image);
+                  }),
+                  title: SearchBox(),
+                  actions: <Widget>[
+                    IconButton(
+                      onPressed: _doLogout,
+                      icon: const Icon(Icons.menu),
+                    )
+                  ]),
+              body: Stack(
+                children: <Widget>[
+                  Positioned.fill(
+                      child: Container(
+                          color: Color(0xFF1c1a34),
+                          child: ListView(
+                            children: <Widget>[
+                              ExpenseImageAsset2(),
+                              Container(
+                                child: Row(
+                                    children: [
+                                  Padding(
+                                      padding:
+                                          EdgeInsets.only(left: 20, top: 71),
+                                      child: Column(children: [
+                                        SampleButton(
+                                            'assets/custom_icons/posts.png',
+                                            'posts'),
+                                        SampleButton(
+                                            'assets/custom_icons/challenges.png',
+                                            'challenges'),
+                                        SampleButton(
+                                            'assets/custom_icons/social-groups.png',
+                                            'social groups'),
+                                        SampleButton(
+                                            'assets/custom_icons/treasure.png',
```

Comment:
bu ikonları bir constant içerisine alıp oradan çağırın. ilerde elle değiştirmek yerine contant içerisinde tek seferde koda müdahale etmeden değiştirebilirsiniz.

NOT: bütün sabitleri mutlaka constant class içerisine alın